nav2_smac_planner/src/collision_checker.cpp

```diff
@@ -86,6 +86,15 @@ bool GridCollisionChecker::inCollision(
   const float & angle_bin,
   const bool & traverse_unknown)
 {
+  // Check to make sure cell is inside the map
+  if (outsideRange(0, costmap_->getSizeInCellsX(), x) ||
+      outsideRange(0, costmap_->getSizeInCellsY(), y))
+  {
+    return false;
+  }
+
+
```

Comment:
Remove extra lines

Reply:
Actually, run the linters ament_cpplint and ament_uncrustify and fix any issues, there are probably more